pallets/ddc-clusters/src/weights.rs

```diff
@@ -31,108 +31,168 @@ pub trait WeightInfo {
 	// Storage: Contracts CodeStorage (r:1 w:0)
+	// Proof: Contracts CodeStorage (max_values: None, max_size: Some(126001), added: 128476, mode: MaxEncodedLen)
 	// Storage: Timestamp Now (r:1 w:0)
+	// Proof: Timestamp Now (max_values: Some(1), max_size: Some(8), added: 503, mode: MaxEncodedLen)
 	// Storage: System EventTopics (r:2 w:2)
+	// Proof Skipped: System EventTopics (max_values: None, max_size: None, mode: Measured)
+	// Storage: DdcClusters ClustersNodesStats (r:1 w:1)
+	// Proof Skipped: DdcClusters ClustersNodesStats (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcClusters ClustersNodes (r:0 w:1)
-	// Storage: unknown [0x89eb0d6a8a691dae2cd15ed0369931ce0a949ecafa5c3f93f8121833646e15c3] (r:1 w:0)
-	// Storage: unknown [0xc3ad1d87683b6ac25f2e809346840d7a7ed0c05653ee606dba68aba3bdb5d957] (r:1 w:0)
+	// Proof Skipped: DdcClusters ClustersNodes (max_values: None, max_size: None, mode: Measured)
+	// Storage: unknown `0x89eb0d6a8a691dae2cd15ed0369931ce0a949ecafa5c3f93f8121833646e15c3` (r:1 w:0)
+	// Proof Skipped: unknown `0x89eb0d6a8a691dae2cd15ed0369931ce0a949ecafa5c3f93f8121833646e15c3` (r:1 w:0)
+	// Storage: unknown `0xc3ad1d87683b6ac25f2e809346840d7a7ed0c05653ee606dba68aba3bdb5d957` (r:1 w:0)
+	// Proof Skipped: unknown `0xc3ad1d87683b6ac25f2e809346840d7a7ed0c05653ee606dba68aba3bdb5d957` (r:1 w:0)
 	fn add_node() -> Weight {
-		Weight::from_parts(599_000_000_u64, 0)
-			.saturating_add(T::DbWeight::get().reads(14_u64))
-			.saturating_add(T::DbWeight::get().writes(5_u64))
+		Weight::from_parts(3_972_000_000_u64, 0)
+			.saturating_add(T::DbWeight::get().reads(15_u64))
+			.saturating_add(T::DbWeight::get().writes(6_u64))
 	}
 	// Storage: DdcClusters Clusters (r:1 w:0)
+	// Proof Skipped: DdcClusters Clusters (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcNodes StorageNodes (r:1 w:1)
-	// Storage: DdcClusters ClustersNodes (r:0 w:1)
+	// Proof Skipped: DdcNodes StorageNodes (max_values: None, max_size: None, mode: Measured)
+	// Storage: DdcClusters ClustersNodes (r:1 w:1)
+	// Proof Skipped: DdcClusters ClustersNodes (max_values: None, max_size: None, mode: Measured)
+	// Storage: DdcClusters ClustersNodesStats (r:1 w:1)
+	// Proof Skipped: DdcClusters ClustersNodesStats (max_values: None, max_size: None, mode: Measured)
 	fn remove_node() -> Weight {
-		Weight::from_parts(24_000_000_u64, 0)
-			.saturating_add(T::DbWeight::get().reads(2_u64))
-			.saturating_add(T::DbWeight::get().writes(2_u64))
+		Weight::from_parts(505_000_000_u64, 0)
+			.saturating_add(T::DbWeight::get().reads(4_u64))
+			.saturating_add(T::DbWeight::get().writes(3_u64))
 	}
 	// Storage: DdcClusters Clusters (r:1 w:1)
+	// Proof Skipped: DdcClusters Clusters (max_values: None, max_size: None, mode: Measured)
 	fn set_cluster_params() -> Weight {
-		Weight::from_parts(16_000_000_u64, 0)
+		Weight::from_parts(244_000_000_u64, 0)
```

Comment:
The weights were taken from Rakan's PR and was executed on a dedicated benchmarking machine